generator/ServiceClientGeneratorLib/Generators/Marshallers/BaseResponseUnmarshaller.cs

```diff
@@ -1680,6 +1680,11 @@ protected string DetermineXmlMarshallName(Member member)
                 return member.Shape.Name;
             return member.MarshallName;
         }
+		if (!withPrefix)
+		{
+			var locationNameString = locationName.ToString();
+			return locationNameString.StartsWith("xsi:") ? locationNameString.Substring(4) : locationNameString;
```

Comment:
Is the prefix always going to be xsi or should we be doing a more generic prefix strip?